### PR TITLE
docs(aio): add missing configuration for angular SSR / angular universal tutorial guide

### PR DESCRIPTION
**Pull request by camilogiraldo**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [N/A ] Tests for the changes have been added (for bug fixes / features)
- [ N/A] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ X] Documentation content changes
```

## What is the current behavior?
Add missing universal tutorial documentation regarding the `.angular-cli.json` and the `main.server.ts`.

Issue Number: #22873

## Does this PR introduce a breaking change?
```
[x ] No
```

### CODE REVIEW

File: aio/content/guide/universal.md [hunk 1]
```diff
@@ -421,6 +429,54 @@ This config extends from the root's `tsconfig.json` file. Certain settings are n
 * The `angularCompilerOptions` section guides the AOT compiler:
   * `entryModule` - the root module of the server application, expressed as `path/to/file#ClassName`.
 
+### Creating a new project in `.angular-cli.json`
```
Comment (Toxicable) [inline aio/content/guide/universal.md]: This is no longer the case in CLI v6

**Comment by brandonroberts** [comment]
Thanks @camilogiraldo. Will you rebase this branch after https://github.com/angular/angular/pull/23842 lands? It has updates for V6 and those missing configuration files also.

**Comment by brandonroberts** [comment]
@camilogiraldo are you still interested in landing this PR?

**Comment by brandonroberts** [comment]
Closing for now. If you're still interested in updating, reopen this PR or open a new one. Thanks

**Pull request closed without merging**